include/mc_rbdyn/RobotConverter.h

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2022 CNRS-UM LIRMM, CNRS-AIST JRL
+ */
+
+#pragma once
+#include <mc_rbdyn/Robot.h>
+#include <mc_rbdyn/api.h>
+
+namespace mc_rbdyn
+{
+
+/**
+ * @brief Configuration for mc_rbdyn::RobotConverter
+ */
+struct RobotConverterConfig
+{
+  ///< Copy input robot's mbc to their corresponding output robot's mbc
+  bool mbcToOutMbc_ = true;
+  /**
+   * Chosse which mbc properties to copy, only effective if mbcToOutMbc_ = true
+   * @{
+   */
+  bool q_ = true;
+  bool alpha_ = true;
+  bool alphad_ = true;
+  bool tau_ = true;
+  ///< @}
+
+  ///< Copy input encoder values to the output robot's mbc
+  bool encodersToOutMbc_ = true;
```

Comment:
I think this is a dangerous default

Reply:
If we do this, and don't copy the mbc afterwards, then yes the behaviour is a bit dangerous (this is technically what the soft emergency on HRP5P does: lower the gains, set target = encoder).
 
But not copying the robot mbc without setting the encoders is even more dangerous, as we'd get all joint commands jump to zero ;)

But if we do as you suggested in your comment below, and split the canonical conversion code in two parts, the first being always under `mc_rtc`'s full control, then we have no issues of user-side option choices.

Reply:
I have changed the default behavior so that the encoders are only copied the first time the converter runs.